controllers/customers.js

```diff
@@ -0,0 +1,48 @@
+const models = require('../models')
+const Customer = require('../models/customer')
+
+
+const handleUploadCustomers = customers => {
+  const customerEmailValidate = customers.map(customer => {
+    const { email } = customer
+    // removes whitespaces and commas
+    const invalidChars = /[,\s]+|[,\s]+/g
+
+    const validatedEmail = email.replace(invalidChars, '')
+
+    return {
+      ...customer,
+      'email': validatedEmail,
+    }
+  })
+
+  Customer.BulkCreate(customerEmailValidate)
+}
+
+const getAllCustomers = async (req, res) => {
+  const customers = await models.Customer.findAll()
+
+  return res.send(customers)
```

Comment:
Nice! However, I am going to recommend a best practice here, and it is  single responsibility principle in programming.
We can see that all of these `controllers` except for **handleUploadCustomers** functions are taking the same parameters `req`, and `res`. These parameters are created in the routes right? By passing `req`, and `res` you are telling the controller that it should know what is the structure of the request, and the response arguments. That is why we se across these controllers that you do 
```
const { id } = req.params
or
const { firstName, lastName, email, phoneNumber } = res.body
```
These controllers should just receive as arguments the parameters required for them to be able to do what is required for them to do.
In this case `getAllCustomers` what it is doing is just returning all the customers right? And it doesn't need any arguments. And the return value should be the customers. Sending the response through the route should be the responsibility of the route. So what I recommend to do for these functions is something like this.

**controller:**
This function is just going to do what the function name describes right? It is going to get all the customers from the DB.
So what we expect the function to return is the list of customers in the database.
```
const getAllCustomers = async () => { // No need to pass a any value because we don't need it in this function
    return await models.Customer.findAll() // We are just returning the list of customers
}
```

**routes**
The routes should be responsible for breaking down the request body, and send a response. In this case for this method it will be something like this.
```
app.get('/customers', (, res) => {
  try {
      const customers = getAllCustomers()
      res.send(customers)
  } catch (e) {
      res.status(400).send(e.message)
  }
})
```

This is just a recommendation for best practice, it will be also easier to maintain the code by doing this refactoring.